src/test/java/net/logstash/logback/encoder/LogstashEncoderTest.java

```diff
@@ -107,43 +103,42 @@ public void basicsAreIncluded_logback12() throws Exception {
 
         when(logback11Support.isLogback11OrBefore()).thenReturn(true);
         final long timestamp = System.currentTimeMillis();
-        
+
         ILoggingEvent event = mockBasicILoggingEvent(Level.ERROR);
         when(event.getTimeStamp()).thenReturn(timestamp);
-        
+
         encoder.start();
         byte[] encoded = encoder.encode(event);
-        
+
         JsonNode node = MAPPER.readTree(encoded);
-        
+
         verifyBasics(timestamp, node);
     }
 
     protected void verifyBasics(final long timestamp, JsonNode node) {
-        assertThat(node.get("@timestamp").textValue()).isEqualTo(FastDateFormat.getInstance("yyyy-MM-dd'T'HH:mm:ss.SSSZZ").format(timestamp));
+        assertThat(node.get("@timestamp").textValue()).isEqualTo(String.valueOf(timestamp));
         assertThat(node.get("@version").textValue()).isEqualTo("1");
         assertThat(node.get("logger_name").textValue()).isEqualTo("LoggerName");
         assertThat(node.get("thread_name").textValue()).isEqualTo("ThreadName");
         assertThat(node.get("message").textValue()).isEqualTo("My message");
         assertThat(node.get("level").textValue()).isEqualTo("ERROR");
         assertThat(node.get("level_value").intValue()).isEqualTo(40000);
     }
-    
+
     @Test
     public void basicsAreIncludedWithShortenedNames() throws Exception {
         final long timestamp = System.currentTimeMillis();
-        
+
         ILoggingEvent event = mockBasicILoggingEvent(Level.ERROR);
-        
+
         when(event.getTimeStamp()).thenReturn(timestamp);
         encoder.setFieldNames(new ShortenedFieldNames());
         encoder.start();
         byte[] encoded = encoder.encode(event);
-        
+
         JsonNode node = MAPPER.readTree(encoded);
-        
-        assertThat(node.get("@timestamp").textValue()).isEqualTo(FastDateFormat.getInstance("yyyy-MM-dd'T'HH:mm:ss.SSSZZ").format
-                (timestamp));
+
+        assertThat(node.get("@timestamp").textValue()).isEqualTo(String.valueOf(timestamp));
```

Comment:
Same comment as above.  Why is the expected value here a long value converted to a string?

Reply:
I missunderstood the StringTimestampWriter, as a result I've done some wrong changes here